advanced/Templates/gravity.db.sql

```diff
@@ -1,6 +1,6 @@
 PRAGMA FOREIGN_KEYS=ON;
 
-CREATE TABLE domain_groups
+CREATE TABLE groups
```

Comment:
We have not been following this practice yet, but usually for tables you want to name them in the singular, ex. "group", "adlist", etc:
https://stackoverflow.com/a/5841297

Reply:
Also, a table with name `group` is not allowed. We will need to use `groups` or something different as `GROUP BY ...` is an SQL instruction.

Reply:
You can wrap it in double quotes. The same would happen if you added a `user` table, which is very common.